core/src/main/java/com/linecorp/armeria/server/annotation/Header.java

```diff
@@ -22,10 +22,26 @@
 import java.lang.annotation.Target;
 
 /**
- * Annotation for mapping an HTTP request header onto a method parameter.
+ * Annotation for mapping an HTTP request header onto the following elements.
+ *
+ * <p>a parameter of an annotated service method
+ *
+ * <p>or, a field of a request bean (See: {@link RequestConverter} and {@link RequestConverterFunction})
```

Comment:
I think we can just leave one `(See: {@link RequestConverter} and {@link RequestConverterFunction})` at the end and remove the others.

Reply:
fixed